core/domain/collection_services.py

```diff
@@ -481,6 +488,64 @@ def get_collection_summaries_matching_ids(collection_ids):
             collection_ids)]
 
 
+def update_collection_node_prerequisite_skills(
```

Comment:
I'm wondering whether we actually need this or ``update_collection_node_acquired_skills``. If we perform the same consistency check as part of ``Collection``'s validation, can't we just use ``CollectionNode``'s corresponding update methods?